lib/matplotlib/pyplot.py

```diff
@@ -1185,6 +1185,11 @@ def close(fig: None | int | str | Figure | Literal["all"] = None) -> None:
         - ``str``: a figure name
         - 'all': all figures
 
+    Note
+    ----
```

Comment:
```suggestion
    Notes
    -----
```
Should fix the doc build

Reply:
oops, fixed.